### ISSUE TITLE
Remove "psurya-jenkins" links

### ISSUE THREAD

**Issue by prakashsurya** [openzfs/openzfs-build#11]
There's a number of places where my internal Jenkins domain name is being leaked via hyperlinks in the public Jenkins UI, as well as in the GitHub pull request status messages that are posted. These should all get updated to link to the public Jenkins instance instead.

**Comment by prakashsurya** [comment]
To fix this, we need to set the private instance's `JENKINS_URL` to the URL of the public instance.

**Comment by prakashsurya** [comment]
I've partially addressed this in the comit `46ae93ef2a0b1213956561dcaf1d1c03be47226b` where I remove the rsync'd file that specifies the `JENKINS_URL`. This should allow the public instance to use its own hostname as the `JENKINS_URL` and get the links to correctly reference the public instance.

The last remaining bit is to update the pull request trigger of the `openzfs-regression-tests` job to properly set the `statusUrl` field for the commit statuses. Right now, the "Details" button of the tests of pull requests will link back to the private instance. If we could se the `statusUrl` we could have that button link to `http://build.prakashsurya.com:8080/job/${JOB_NAME}/${BUILD_NUMBER}/` .. Unfortunately we can't do this in the Jenkins DSL until this commit lands in a release: https://github.com/jenkinsci/job-dsl-plugin/pull/633

Until we can set this in the DSL, we have to manually set this in the Jenkins GUI and be careful not to blow this manual configuration away by re-running the "seed-job" (each time the seed-job is run, this manual configuration needs to occur as well).

**Comment by prakashsurya** [comment]
And just to be more explicit about what I mean, here's a patch that I think would be sufficient to close this issue, but I can't until the Jenkins DSL supports this:

```
commit 826e5d237bc37e561bf23ef7c0e21268419f2087
Author: Prakash Surya <prakash.surya@delphix.com>
Date:   Wed Oct 21 23:59:44 2015 -0700

    Use the public Jenkins URL for commit statuses

    NOTE: Unfortunately, the statusUrl isn't exposed in the pull request
    builder plug-in yet. This was added in this commit:

        https://github.com/jenkinsci/job-dsl-plugin/commit/bc488a599b474f8917db301d624fab9db36c1131

    but this hasn't yet landed in a tagged release of the plug-in.

    This is a small patch to update the GitHub pull request builder plug-in
    to use the public Jenkins instance URL for the commit status URLs. This
    will point outside contributers directly to the public instance, instead
    of the internal one.

diff --git a/jobs/openzfs_regression_tests.groovy b/jobs/openzfs_regression_tests.groovy
index 317d87c..6b6029d 100644
--- a/jobs/openzfs_regression_tests.groovy
+++ b/jobs/openzfs_regression_tests.groovy
@@ -1,3 +1,5 @@
+def url = 'http://build.prakashsurya.com:8080'
+
 /*
  * Orchestrate a full build and test cycle of OpenZFS, including the
  * creation and destruction of Jenkins slaves to do the heavy lifting.
@@ -108,6 +110,7 @@ multiJob("openzfs-regression-tests") {
                     triggeredStatus('Tests have been triggered.')
                     completedStatus('SUCCESS', 'Tests passed.')
                     completedStatus('FAILURE', 'Tests failed.')
+                    statusUrl(url + '/job/${JOB_NAME}/${BUILD_NUMBER}/')
                 }
             }
         }
```
